libs/domain/search/facet/facet.styles.ts

```diff
@@ -21,6 +21,7 @@ export const searchFacetStyles = css`
 
   input + div {
     display: flex;
+    justify-content: space-between;
```

Comment:
there is wrong rendering in the case when there is no counter for facet value. To prevent it I changed styles, so now is correct rendering in both cases - with counter and without counter